airflow/providers/amazon/aws/operators/glue.py

```diff
@@ -100,6 +101,7 @@ def __init__(
         verbose: bool = False,
         update_config: bool = False,
         job_poll_interval: int | float = 6,
+        stop_job_run_on_kill: bool = False,
```

Comment:
Can you add a `:param stop_job_run_on_kill: ...` description to the method doc string please?